### PR TITLE
Use `superagent-legacyIESupport` plugin

### PR DESCRIPTION
**Pull request by iest**
Adds cross-domain legacy support to IE9

### CODE REVIEW

File: modules/PactResource.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import request from 'superagent';
+import lolIE from 'superagent-legacyIESupport';
```
Comment (rdjpalmer) [inline modules/PactResource.js]: lolNaming

File: test/PactResource-test.js [hunk 2]
```diff
@@ -126,6 +126,8 @@ describe('PactResource', () => {
     it('Sends the payload if there is one', () => {
       const payload = {foo: 'bar'};
       instance._request('get', '/testpath', payload);
+      console.log('HEYYY');
```
Comment (rdjpalmer) [inline test/PactResource-test.js]: Unnecessary log

**Comment by rdjpalmer** [comment]
You sticking with `lolIE`? :ship:

**Comment by iest** [comment]
You don't like it?

**Comment by rdjpalmer** [comment]
Not overly, but its no problem :grin:

**Pull request merged**